sample/src/main/java/com/kwarchart/sample/ui/bar/BarFragment.kt

```diff
@@ -4,21 +4,26 @@ import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.widget.AdapterView
+import android.widget.ArrayAdapter
 import android.widget.Button
+import android.widget.Spinner
 import androidx.compose.foundation.background
 import androidx.compose.foundation.layout.fillMaxHeight
 import androidx.compose.foundation.layout.fillMaxWidth
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.platform.ComposeView
 import androidx.fragment.app.Fragment
-import androidx.lifecycle.Observer
 import androidx.lifecycle.ViewModelProvider
 import com.kwarchart.android.chart.BarChart
 import com.kwarchart.android.enum.LegendPosition
 import com.kwarchart.sample.databinding.FragmentBarBinding
 
-class BarFragment : Fragment() {
+
+
+
```

Comment:
remove extra line breaks

Reply:
Let me handle this one... Will fix it in HORIZONTAL BarChart PR.